cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,108 @@
+/// <reference types='cypress' />
+
+import { generateWorker } from '../support/generateWorker'
+
 describe('Web Tables page', () => {
-  it('', () => {
-    
+  const { firstName, lastName, email, age, salary, department } = generateWorker();
+  
+  beforeEach (() => {
+    cy.visit('/');
+  });
+
+  it('should contain pagination', () => {
+    cy.get('.-pageJump')
+      .should('exist');
+
+    cy.get('.-previous > .-btn')
+      .should('exist')
+      .should('contain.text', 'Previous');
+
+    cy.get('.-next > .-btn')
+      .should('exist')
+      .should('contain.text', 'Next');
+  });
+
+  it('should contain rows count selection', () => {
+    const randomRowIndex = Math.floor(Math.random() * 6);
+    const rows = ['5 rows', '10 rows', '20 rows', '25 rows', '50 rows', '100 rows']
+    const row = rows[randomRowIndex];
+
+    cy.get('select')
+      .select(row);
```

Comment:
But you don't test anything, where are asserts?

Reply:
I test the availability of the UI elements.

Reply:
Well, I think that "should contain rows count selection" means that I can choose any item in the dropdown menu which is exactly what I test here.